plutus-tx/src/PlutusTx/Prelude.hs

```diff
@@ -139,6 +140,19 @@ check b = if b then () else traceError checkHasFailedError
 divide :: Integer -> Integer -> Integer
 divide = Builtins.divideInteger
 
+{-# INLINABLE exponent #-}
+-- | Integer exponentiation
+--
+--   >>> exponent 5 2
+--   25 
+--
+exponent :: Integer -> Integer -> Integer
```

Comment:
I think this is confusing since the `exponent` function in the prelude does something different.

If we're giving it a non-symbolic name then I'd use the verb, i.e. `exponentiate`.